public/app/core/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
-import { mount, ReactWrapper } from 'enzyme';
 import { useSelector } from 'react-redux';
 import { dataQa } from '@percona/platform-core';
-import { act } from 'react-dom/test-utils';
 import { Breadcrumb } from './Breadcrumb';
+import { generateMountWrapper } from 'app/percona/shared/helpers/testUtils';
 
-jest.mock('react-redux', () => ({
-  ...jest.requireActual('react-redux'),
-  useSelector: jest.fn(),
-}));
+jest.mock('react-redux', () => {
```

Comment:
The version above was not that bad, I guess. The point is, I wouldn't refactor what doesn't seem to need to be refactored.

Reply:
Our linter will complain about it, that's why I had to change it

Reply:
Specifically:
`Spread types may only be created from object types`